DorisProject/src/app/pages/stock/stock.component.ts

```diff
@@ -61,6 +61,12 @@ export class StockComponent implements OnInit {
         if (product.Category.toLowerCase().includes(event.target.value.toLowerCase())) {
           SearchList.push(product);
         }
+        if (product.Name.toLowerCase().includes(event.target.value.toLowerCase())) {
+          SearchList.push(product);
+        }
+        if (product.Brand.toLowerCase().includes(event.target.value.toLowerCase())) {
```

Comment:
Codacy has a fix for the issue: [Replace `product.Brand.toLowerCase().includes(event.target.value.toLowerCase())` with `⏎··········product.Brand.toLowerCase().includes(event.target.value.toLowerCase())⏎········`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9283869)
```suggestion
        if (
```